client/my-sites/post-type-list/post-actions-ellipsis-menu/publish.jsx

```diff
@@ -78,15 +78,17 @@ const mapStateToProps = ( state, { globalId } ) => {
 	};
 };
 
-const mapDispatchToProps = { savePost, bumpAnalyticsStat };
+const mapDispatchToProps = { savePost, bumpStat, recordTracksEvent };
 
 const mergeProps = ( stateProps, dispatchProps, ownProps ) => {
-	const bumpStat = bumpStatGenerator(
-		get( stateProps, 'type.name' ),
-		'publish',
-		dispatchProps.bumpAnalyticsStat
+	const postType = get( stateProps, 'type.name' );
+	const onPublishPost = () => (
+		bumpStatGenerator( postType, 'publish', dispatchProps.bumpStat ),
```

Comment:
This PR broke collection of a few MC stats (duplicate, trash, delete) because `bumpStatGenerator` [returns a function](https://github.com/Automattic/wp-calypso/blob/21898005ef7a965b1a2037eb85e2081748f61518/client/my-sites/post-type-list/post-actions-ellipsis-menu/utils.js#L2) which needs to be called to actually bump the stat.

I would also recommend against using the `( statement1, statement2 )` syntax as it seems like it could be confusing and potentially introduce bugs later.

I'm fixing these issues along with the merge of #19625 because there were conflicts in this specific area of the code.